### ISSUE TITLE
Please do a HTML export

### ISSUE THREAD

**Issue by JuergenSchuster** [OraOpenSource/plsql-md-doc#37]
Hi guys, 

I know you are living in a tech world and find markup cool, but me and my customer actually looking for a PL/SQL documentation tool like PL/DOC where I simply can generate an HTML file for documentation where the normal people like my customers and me :-) can simply invoke it with any browser. I would use PL/DOC but since Martin did that great job with Logger which we use heavily and PL/DOC looks quite old I'm hoping for a similar new standard solution for PL/SQL documentation from him. Let me assure you this is the only feature that separates you from a serious standard product in the PL/SQL world ;-)

Thanks, Juergen

**Comment by vincentmorneau** [comment]
The name of this project (plsql-md-doc) is almost misleading, because it doesn't just do markdown.

If you look into the [template](https://github.com/OraOpenSource/plsql-md-doc/tree/master/templates) folder, you'll see `package.md` and `toc.md`. @martindsouza created default markdown templates, but these could be changed to generate HTML files as well.

This project uses handlebars to substitute the content of your PL/SQL doc into any file. For instance, the `toc.md` looks like:

``` handlebars
# {{toUpperCase projectDispName}}

{{#each files}}
- [{{toUpperCase name}}]({{fileName}})
{{/each}}
```

But you could also create your own to look like:

``` handlebars
<h1> {{toUpperCase projectDispName}} </h1>

<ul>
{{#each files}}
<li><a href="{{fileName}}">{{toUpperCase name}}</a></li>
{{/each}}
</ul>
```

I do agree it would be useful to provide a "customer" readable template out of the box.

**Comment by martindsouza** [comment]
We must produce a good looking HTML file, not just basic. @vincentmorneau what do you think about using a Bootstrap CDN to help with the HTML markup? This way you can get something that actually looks good? http://getbootstrap.com/getting-started/

Also: http://usejsdoc.org/about-getting-started.html

**Comment by vincentmorneau** [comment]
Yes I think that's a good idea. If you want I can help. Just assign me to this issue and I'll find some time soon to produce a quality HTML template.

**Comment by JuergenSchuster** [comment]
I guarantee you nobody cares about all the hard work in the backend, if you provide a nice, modern, responsive HTML Frontend. Everybody will like it and use it :-)

**Comment by JuergenSchuster** [comment]
I would call it Docer to have a famous pair that can benifit from each other: Logger and Docer ;-) This sounds like a strong couple you should both use in your projects :-)

**Comment by martindsouza** [comment]
@JuergenSchuster @vincentmorneau #48 may resolve this.

**Comment by vincentmorneau** [comment]
https://github.com/OraOpenSource/plsql-md-doc/pull/56 resolved this issue.
